sematic/config/server_settings.py

```diff
@@ -32,6 +37,9 @@ class ServerSettingsVar(AbstractSettingsVar):
     # GRAFANA
     GRAFANA_PANEL_URL = "GRAFANA_PANEL_URL"
 
+    # Server-side plug-ins section
+    plugins = "plugins"
```

Comment:
This new entry does not fit in with the other setting enums. The CLI integration will not properly work anymore.

The consequence will be that you will be able to do `sematic settings get plugin`, and it would would pass validation and return the entire blob of configurations.

You will be able to overwrite the setting with anything using `sematic settings set plugin anything`, and the error will only come when trying to use these settings inside the application.

Reply:
Good point, I will remove the plugins key from the enum and treat it separately so that it is not exposed to the CLI.